security-check.php

```diff
@@ -0,0 +1,8 @@
+<?php
+if (!defined('ABSPATH')) {
+    // Exit if accessed directly
+    exit;
+}
+
+// Do not display any error
+ini_set('display_errors', 0);
```

Comment:
Might be worth just not setting this at all, and let the server decide

Reply:
I was thinking of exclusively disabling all the errors from showing but I guess it does override the rule from the server. Given we have the path check, it won't hit the error anyway so I will remove it.